src/components/pages/MobileNavMenu/styles.tsx

```diff
@@ -14,5 +14,5 @@ export const StyleMobileNavMenuWrapper = styled.div<{ open: boolean }>`
     top: 0;
     transform: ${props => props.open ? 'translateX(0)' : 'translateX(-110%)'};
     transition: transform 0.5s ease-in-out;
-    z-index: ${zindex.overlay}
+    z-index: ${zindex.menu}
```

Comment:
do you need a really high `zindex`? `zindex.alpha` or `zindex.beta` are not enough?

Reply:
I thought it would be best to save those in case they were needed somewhere else or could conflict with the menu. If that's not really a problem I can just use alpha/beta etc.